BMM.Core/App.cs

```diff
@@ -262,6 +265,13 @@ public override void Initialize()
             RegisterDynamic(typeof(IActionExceptionHandler));
         }
 
+        private static void SetupLanguageBinder()
+        {
+            var languageBinder = new BMMLanguageBinder();
+            BMMLanguageBinderLocator.SetImplementation(languageBinder);
+            Mvx.IoCProvider.RegisterSingleton(typeof(IBMMLanguageBinder), new BMMLanguageBinder());
```

Comment:
shouldn't you return the languageBinder variable? Now 2 instances are created

Reply:
You're right, my mistake